lib/bee/loader/Loader.rb

```diff
@@ -8,7 +8,7 @@ def initialize(fname, config)
       beedb = config.get(:beedb)
       @writer = beedb ?
         Bee.const_get(config.get(:writer)).new(beedb) :
-        Bee.const_get(config.get(:writer)).new
+        Bee.const_get(config.get(:writer)).new("",config.get(:label))
```

Comment:
If the "beedb" key is provided, this indicates an embedded DB and the value of the key is passed on line 10, without label.

If the key is not provided, we have a non-embedded DB, so an empty DB name is provided on line 11 with the configuration file's "label" entry as label name.